docs/reference/installation.rst

```diff
@@ -68,20 +68,6 @@ To use the ``BlockBundle``, add the following lines to your application configur
         sonata_block:
             default_contexts: [sonata_page_bundle]
             blocks:
-                sonata.admin.block.admin_list:
```

Comment:
https://github.com/symfony/recipes-contrib/blob/master/sonata-project/admin-bundle/3.31/config/packages/sonata_admin.yaml

should probably be changed here also